src/include/cpu/power/spr.h

```diff
@@ -14,6 +14,7 @@
 #define SPR_CIABR				0xBB
 #define SPR_DAWRX				0xBC
 #define SPR_TB					0x10C
+#define SPR_PVR					0x11F
```

Comment:
One TAB too many, perhaps?

Reply:
Nope, GitHub has problems with rendering, it treats `+` as a separate character. Looks correct when file is opened without diff: https://github.com/3mdeb/coreboot/blob/630b8e04e4f7b3ddb3bbce8bf1c6ddc40c5205a4/src/include/cpu/power/spr.h